nemo_skills/inference/merge_chunks.py

```diff
@@ -25,13 +39,21 @@
 
 # Concatenate the files using subprocess
 try:
-    subprocess.run(["cat"] + input_files, stdout=open(output_file, "w"), check=True)
+    with open(output_file, "w") as out_f:
+        subprocess.run(["cat"] + input_files, stdout=out_f, check=True)
     print(f"Successfully concatenated {len(input_files)} files to {output_file}")
 
     # Create .done file and delete input files
     open(f"{output_file}.done", "w").close()
     for file_ in input_files:
         os.remove(file_)
-except subprocess.CalledProcessError:
-    print("An error occurred while concatenating files")
+
+    # Execute the post-merge command, if provided
+    if post_merge_command:
+        print(f"Executing post-merge command: {' '.join(post_merge_command)}")
+        result = subprocess.run(' '.join(post_merge_command), shell=True, check=True, capture_output=True, text=True)
```

Comment:
do we need to capture output? What if some important message is in stderr, is that going to be printed or hidden now?

Reply:
you're right--it's better to not have the capture  logic. It logs it all fine with the `check` and `capture_output` flags set back to default.